React-Native/TodoList/src/app/screens/Main/components/Footer/styles.js

```diff
@@ -1,13 +1,15 @@
 import { StyleSheet } from 'react-native';
+import { LIGHTBLUE, RED } from '../../../../../constants/colors';
```

Comment:
is ok.. as convention we have colors in lower case.. But it's not a problem, lets leave it like this here

Reply:
oh... i get it